popup.js

```diff
@@ -17,7 +17,7 @@ function register(){
 	else
 		set3='';
 
-	window.open("http://www.google.com/webhp?#q="+query+" -inurl:(htm|html|php|pls|txt) intitle:index.of \"last modified\" ("+set1+set2+set3+")&btnI=I");
```

Comment:
Thanks a lot for the explanation. I am new to web development and very
happy for the help i am receiving from you.
On Aug 28, 2016 13:46, "Sparsh Paliwal" <notifications@github.com> wrote:

> In popup.js
> <https://github.com/PaliwalSparsh/chrome-search-extension/pull/1#discussion_r76528038>
> :
>
> > @@ -17,7 +17,7 @@ function register(){
> >  	else
> >  		set3='';
> >
> > -	window.open("http://www.google.com/webhp?#q="+query+" -inurl:(htm|html|php|pls|txt) intitle:index.of \"last modified\" ("+set1+set2+set3+")&btnI=I");
>
> So the the most important part of the whole url was *intitle:index.of*.
> This searches across internet for all the pages with *index of* in the
> title, almost all the ftp servers have *"index of"* in title. After this
> all the ftp pages have a *last modified* element on the webpage. So this
> part of the query ensure that the page google searches is a ftp server
> only.
>
> Here we are searching across ftp servers only because people keep a lot of
> data on their ftp server, and there are thousands of ftp servers. So you
> can find movies, songs, documents easily and most importantly for free.
> Though the problem that was with this particular extension was that most of
> the ftp servers are password protected, so this is an issue for many
> queries.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/PaliwalSparsh/chrome-search-extension/pull/1/files/8d7c0d11d240778c3db24aa65616f35bf860eaf9#r76528038>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AUTD2HqKb26RJBqrgU0JeTCVyjwd7bIAks5qkUP5gaJpZM4Ju3gJ>
> .
>